src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+public class Ball extends Application{
+    private String color;
+    private int number;
+    public Ball (ColorsEnum color, int number) {
```

Comment:
should always be empty line between fields and methods/constructor, and between methods themselves